src/home_assistant_bluetooth/models.py

```diff
@@ -1,10 +1,12 @@
 """The bluetooth integration service info."""
 
-from typing import Any, Dict, Final, List, Optional, Type, TypeVar
+from typing import TYPE_CHECKING, Any, Dict, Final, List, Optional, Type, TypeVar
 
 from bleak.backends._manufacturers import MANUFACTURERS
```

Comment:
This import still requires `bleak`, so it cannot be just dev-dependency.

Reply:
fixed in https://github.com/home-assistant-libs/home-assistant-bluetooth/pull/17